src/storage.ts

```diff
@@ -371,6 +374,13 @@ export class Storage extends Service {
    * @property {boolean} [autoRetry=true] Automatically retry requests if the
    *     response is related to rate limits or certain intermittent server
    * errors. We will exponentially backoff subsequent requests by default.
+   * @property {number} [backoffMultiplier = 2] Dictates the rate of exponential backoff.
```

Comment:
I think it might be clearer if you add a little explanation/example about how these parameters interact with one another. For example with the default settings, up to how many retries will happen and how much will they be spaced out? You could also link to something explaining backoff if that's easier.